### ISSUE TITLE
Coverage is including fixtures

### ISSUE THREAD

**Issue by jradtilbrook** [HealthEngineAU/he-react-ui#1]
Looks like the test coverage is including fixture files. Aren't these just data files for use by cosmos to show different states? Therefore shouldn't be included in the coverage?

**Comment by xow** [comment]
Makes sense to me. I think CSS might not be working properly, perhaps we should remove that too. I need to double check

**Comment by xow** [comment]
```
"collectCoverageFrom": [
      "src/**/*.{js,jsx}",
      "!src/**/*.fixture.js"
    ]
```

**Comment by xow** [comment]
Fixed in f6bc9e3
